### PR TITLE
:rocket: deploy-conda: PR to deploy to conda-forge channel

### PR DESCRIPTION
**Pull request by tngTUDOR**
🤖

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by tngTUDOR** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (m-rossi) [inline recipe/meta.yaml:1]: Please update requirements matchting to https://github.com/brightway-lca/brightway2-data/blob/4.0.dev53/pyproject.toml

**Review by m-rossi — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml
Comment (tngTUDOR) [reply recipe/meta.yaml:1]: ✅

Comment (tngTUDOR) [reply recipe/meta.yaml:1]: + lint

**Comment by tngTUDOR** [comment]
the checks fail with:

```
Adding in variants from internal_defaults
Adding in variants from /home/conda/recipe_root/conda_build_config.yaml
Adding in variants from /home/conda/feedstock_root/.ci_support/linux_64_.yaml
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/conda/models/version.py", line 44, in __call__
    return cls._cache_[arg]
KeyError: "sys_platform!='emscripten'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/conda/models/version.py", line 44, in __call__
    return cls._cache_[arg]
KeyError: "sys_platform!='emscripten'"

```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1027283&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=514

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ requirements:
     - peewee >=3.9.4
     - platformdirs
     - pydantic-settings
-    - rapidfuzz; sys_platform != 'emscripten'
+    - rapidfuzz
```
Comment (michaelweinold) [inline recipe/meta.yaml:35]: Correct - I have my own recipes for now.

**Comment by tngTUDOR** [comment]
@michaelweinold would it affect bwlive to have this package published without the "sys_platform != emscipten" constraint ?

**Comment by michaelweinold** [comment]
Thank you for checking regarding Brightway Live!
You're good to edit these recipes at will. I am currently using my own set of recipes.

**Comment by m-rossi** [comment]
We would add such a restraint differently here on conda-forge. But it is not necessary right now as the packages are not build for emscripten right now

**Comment by michaelweinold** [comment]
> But it is not necessary right now as the packages are not build for emscripten right now

It is not directly relevant whether they are "built for emscripten".

The way in which I add packages to an emscripten-based JupyterLite instance is through regular conda-forge packages - as long as they are pure python, or dependencies are on emscripten-forge.

In any case, all emscripten-related points don't touch in the updates to the Brightway packages you are undertaking right now.

**Comment by m-rossi** [comment]
> The way in which I add packages to an emscripten-based JupyterLite instance is through regular conda-forge packages - as long as they are pure python, or dependencies are on emscripten-forge.

I was not aware of that. Thought you are using https://emscripten-forge.org/

**Comment by michaelweinold** [comment]
No, that is only for packages which are not pure Python. You can see it [in the recipe I am using here](https://github.com/brightway-lca/brightway-live/blob/emscripten-forge/environment.yml).

**Review by m-rossi — APPROVED** [review]

**Comment by cmutel** [comment]
Thanks a lot @m-rossi for your help, and @tngTUDOR @michaelweinold for your contributions. I am really reluctant to add the `dev` branch to conda-forge. For the last few years, the `dev` branch for `bw2io`, `bw2calc`, and `bw2data` have only been on my personal channel. I think that we do need to mark the current code as the main branch (i.e. brightway v2 is deprecated), and are updating the Activity Browser to help the community come along. But I am going to close this PR (while keeping the changes in mind); we an announce that these core libraries will no longer be `dev` as of October 25 (1 month after Brightcon announcement), and then put them all on conda-forge.

**Comment by tngTUDOR** [comment]
@cmutel got it ;)

**Pull request closed without merging**